docs/istio.md

```diff
@@ -481,71 +517,109 @@ Now, open a new browser tab and go to http://localhost:3000 to open Grafana's ho
 
 ![Architecture](../static/images/grafana_1_home.png)
 
-Click on the drop-down next to `Home` and click on the `Istio Service Dashboard` (as shown below) to examine networking requests for the individual services.
+Click on the drop-down next to `Home` and click on the `Istio Service Dashboard` (as shown below) to examine networking
+requests for the individual services.
 
 ![Architecture](../static/images/grafana_2_dashboard_select.png)
 
-On the service drop-down at the top left, select `web.default.svc.cluster.local` service. You should be presented with a dashboard similar to the following:
+On the service drop-down at the top left, select `web.default.svc.cluster.local` service. You should be presented with a
+dashboard similar to the following:
 
 ![Architecture](../static/images/grafana_3_web_dashboard.png)
 
-I'll leave it up to you to examine all the dashboards. The main thing I want to point out is the `Incoming Requests by Source And Response Code` dashboard at the bottom left. The load you see is generated from our script. If you were to interrupt the script, you will see, upon refreshing, that the load will go down to 0. Also notice that the source for all of these requests come directly from the `istio-ingressgateway.istio-system` since we are running our load generation script against the gateway itself instead of through a port-forwarding connection.
+I'll leave it up to you to examine all the dashboards. The main thing I want to point out is the
```

Comment:
Possibly a future change: we might want to rewrite to not use first person.
No change requested for now